fluent-bit-cloudwatch.go

```diff
@@ -175,10 +182,35 @@ func FLBPluginFlushCtx(ctx, data unsafe.Pointer, length C.int, tag *C.char) int
 	return output.FLB_OK
 }
 
-//export FLBPluginExit
+// FLBPluginExit is exported for fluent-bit.
 func FLBPluginExit() int {
 	return output.FLB_OK
 }
 
+func setupLogger() {
```

Comment:
I'm not going to leave this here. I opened https://github.com/aws/amazon-kinesis-firehose-for-fluent-bit/pull/59 for this instead. I'll commit a change to pull this out of here and put this particular setup back how it was. This is where I will test this change again and record the _output_ requested in the linked open PR.